pkg/summarizer/summary_test.go

```diff
@@ -159,7 +165,7 @@ func TestUpdateDashboard(t *testing.T) {
 				}
 				return &fake.group, reader, nil
 			}
-			actual, err := updateDashboard(context.Background(), tc.dash, finder)
+			actual, err := updateDashboard(context.Background(), tc.dash, finder, realClock)
```

Comment:
I don't think you're testing time here, just in getHealthinessForInterval, so I think you only need to pass in time/clock for that function and not for the higher-level ones?

Reply:
I agree that the only place time is tested/needs to be tested is in getHealthinessForInterval. I guess when I put it in the higher level functions I was thinking that we would need to test how getHealthinessForInterval operates within the flow of updateDashboard and updateTab. Is it sufficient to test that getHealthinessForInterval works in a vacuum and reasonably assume it works in the functions that use it? I did think it was messy introducing another parameter just to pass it all the way down to one function.

Reply:
After an offline discussion with @krzyzacy, I'll make a future PR that refactors things so it's easier to test the shouldRunHealthiness block of updateTab. It would involve moving most of the functions in summarizer.go into a summarizer struct that holds members that we can easily inject dummy versions into. 

I took out the new parameter and made getHealthinessForInterval depend on a Time instead of a function that returns Time.